students/rob_sanchez/Lesson 2/Fizz_Buzz.py

```diff
@@ -0,0 +1,16 @@
+def FizzBuzz():
+    #variables
+    num1=3
+    num2=5
+    range_start=1
+    range_end=101
```

Comment:
Please run your code through a [Pep8 linter](http://pep8online.com), or configure your text editor to catch pep8 violations (there are instructions for how to do this for SublimeText in the course materials). Pep8 is the Python style guide and includes many conventions for making your code more readable.